config/locales/es.yml

```diff
@@ -221,14 +222,9 @@ es:
       recommended: recomendado
       title: Notificación de email
       update: Actualizar
-  profiles:
+  settings:
     edit:
-      change_avatar:
-      email_awaiting_confirmation: Por favor confirma tu nueva dirección de correo %{unconfirmed_email}
-      enter_password: Por favor introduce tu contraseña
-      hide_email: Ocultar correo electrónico en perfil público
-      optional_twitter_username: Usuario de Twitter opcional. Será mostrado en tu perfil público
-      title: Editar perfil
+      title:
```

Comment:
`title: Editar configuración.`

Reply:
What do you think about the one on the drop down menu from the nav bar? Should it also be `Editar configuración` or is it too long? Maybe just `Configuración` so it doesn't take up that much space?

Reply:
I think just `Configuración` would work 👍